tests/modules/multiqc/main.nf

```diff
@@ -2,15 +2,31 @@
 
 nextflow.enable.dsl = 2
 
-include { FASTQC  } from '../../../modules/fastqc/main.nf'
-include { MULTIQC } from '../../../modules/multiqc/main.nf'
+include { FASTQC            } from '../../../modules/fastqc/main.nf'
+include { FASTQC as FASTQC2 } from '../../../modules/fastqc/main.nf'
+include { MULTIQC           } from '../../../modules/multiqc/main.nf'
 
 workflow test_multiqc {
     input = [ [ id: 'test', single_end: false ],
-              [ file(params.test_data['sarscov2']['illumina']['test_1_fastq_gz'], checkIfExists: true),
-                file(params.test_data['sarscov2']['illumina']['test_2_fastq_gz'], checkIfExists: true) ]
+                [ file(params.test_data['sarscov2']['illumina']['test_1_fastq_gz'], checkIfExists: true)]
```

Comment:
Would you mind formatting this too please?

Reply:
done!